### PR TITLE
Add clerk-authentication example

### PR DESCRIPTION
**Pull request by SokratisVidros**

### CODE REVIEW

File: edge-functions/clerk-authentication/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https://github.com/vercel/examples/tree/main/edge-functions/clerk-authentication&env=NEXT_PUBLIC_CLERK_FRONTEND_API,CLERK_API_KEY,CLERK_JWT_KEY&project-name=clerk-authentication&repo-name=clerk-authentication)
+
+## Getting Started
+
+You'll need to have an account with [Clerk](https://clerk.dev/?utm_source=vercel&utm_medium=github&utm_campaign=edge-authentication). Once that's done, copy the `.env.example` file in this directory to `.env.local` (which will be ignored by Git):
+
+```bash
+cp .env.example .env.local
+```
+
```
[the remark this answers is not in the document]
Comment (SokratisVidros) [reply edge-functions/clerk-authentication/README.md:30]: Done.

File: edge-functions/clerk-authentication/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "@headlessui/react": "^1.4.1",
+    "@heroicons/react": "^1.0.4",
+    "@peculiar/webcrypto": "^1.1.7",
+    "add": "^2.0.6",
+    "jose": "^3.20.1",
+    "next": "11.1.3-canary.104",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "swiper": "^7.0.9",
+    "yarn": "^1.22.17"
```
Comment (leerob) [inline edge-functions/clerk-authentication/package.json:22]: Is this supposed to be here?

File: edge-functions/clerk-authentication/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@clerk/nextjs": "^2.0.2",
+    "@headlessui/react": "^1.4.1",
+    "@heroicons/react": "^1.0.4",
+    "@peculiar/webcrypto": "^1.1.7",
+    "add": "^2.0.6",
+    "jose": "^3.20.1",
+    "next": "11.1.3-canary.104",
```
Comment (leerob) [inline edge-functions/clerk-authentication/package.json:18]: Could you update all deps? 🙏

File: edge-functions/clerk-authentication/tailwind.config.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  purge: [
```
Comment (leerob) [inline edge-functions/clerk-authentication/tailwind.config.js:2]: Should this use JIT?

File: edge-functions/clerk-authentication/vercel.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "build": {
+    "env": {
+      "FORCE_RUNTIME_TAG": "canary"
```
Comment (leerob) [inline edge-functions/clerk-authentication/vercel.json:4]: This is no longer needed!

Comment (SokratisVidros) [reply edge-functions/clerk-authentication/vercel.json]: Removed.

File: edge-functions/clerk-authentication/tailwind.config.js (hunk 4 above)
Comment (SokratisVidros) [reply edge-functions/clerk-authentication/tailwind.config.js:3]: Added.

File: edge-functions/clerk-authentication/package.json (hunk 3 above)
Comment (SokratisVidros) [reply edge-functions/clerk-authentication/package.json]: Oops. That's right. Bumped to 12.0.1

File: edge-functions/clerk-authentication/vercel.json [hunk 6]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (leerob) [inline edge-functions/clerk-authentication/vercel.json:1]: This whole file can be deleted 😄

**Review by leerob — COMMENTED** [review]
LGTM after that!

File: edge-functions/clerk-authentication/vercel.json (hunk 6 above)
Comment (SokratisVidros) [reply edge-functions/clerk-authentication/vercel.json]: Done.

**Review by leerob — APPROVED** [review]

**Pull request merged**